### ISSUE TITLE
Problem with parser text in menu/page with Polish tag and html v2.0 Bootstrap

### ISSUE THREAD

**Issue by Fiedo** [e107inc/e107#847]
**Hi!**

I have a problem with creating menus in e107 2.0 bootstrap.
When I'm creating a menu with html tags, my text turns into bushes (Polish text with special tags for this language)
But when I'm adding only text (without html tag's)- everything is ok.

I tried with different settings in the menu:
**1. WYSIWIG ON / OFF**
**2. Filter HTML content: enabled / disabled**
**3. Abuse HTML filter (experimental): enabled / disabled**
**No changes! as if the field does NOT work.**
How to disable the parser ??

for example:

*1. First entered the body tags in the menu:*
```
[html]<div id = 'some_div'> <a href='http://sample.html'> Witam zarząd portalu społecznościowego </a> social networking </ div>[/html]
```

*2. The menu after saving:*

```
[html]<div id="some_div"><a href="http://sample.html">Witam zarzÄd portalu spoÅecznoÅciowego</a></div>[/div]
```
In the previous version of the system everything worked.

**I checked the database encoding, screen below:**

![image](http://www.gawron-fiedorow.pl/adam_mamrot_www/problem/baza_danych.jpg)

This coding i think is ok.
I did a few screen with a detailed description of the problem.

**1. My code which I paste in menu/page:**
![image](http://www.gawron-fiedorow.pl/adam_mamrot_www/problem/screen_wpisuje.jpg)

**2. After approved, the menu on the page looks like this:**
![image](http://www.gawron-fiedorow.pl/adam_mamrot_www/problem/strona_po_wpisaniu.jpg)

as you can see in the **MENU TITLE** characters are displayed correctly.
**BODY MENU** after placing html tags appear in the bushes. 
If I use bbcode, for example:
```[html][a href = "some link"] ąśćńó [/ a][/html] ```
characters are displayed correctly.

**3. This menu in the database looks like this:**
![image](http://www.gawron-fiedorow.pl/adam_mamrot_www/problem/baza_danych_po_wpisaniu.jpg)

**menu_title** and **menu_text** have bushes but only **menu_text** is display incorrect. 

**4. I try to paste my code from hand in this place: **
![image](http://www.gawron-fiedorow.pl/adam_mamrot_www/problem/baza_danych_wpisane_reka.jpg)

**5. Result:**
![image](http://www.gawron-fiedorow.pl/adam_mamrot_www/problem/strona_po_zmianie_z_reki_w_bazie.jpg)

When I add some html text into **Welcome message** in admin menu, for example:
```<div class='features' id='features'><div class='container'><div class='features-head text-center'>
		<h2>ąęśćńłó</span></h2> etc.```

everything is displayed correctly... I have this problem only in page/menu tab.

Please help!
Thank you very much!

**Comment by CaMer0n** [comment]
Does your e107_config.php file contain this?
$mySQLcharset   = 'utf8';

**Comment by Jimmi08** [comment]
Only note, I use utf8_general_ci, not utf_unicode_ci. Couldn't be problem there?

**Comment by Fiedo** [comment]
thanks for answer.
CaMer0n: my e107_config.php doesn't contain this line of code, but when i add this line bushes appeared everywhere on the web: welcome message, pages / menus and not only in menu_text but also in menu_title.

Jimmi08 : After installation e107, encoding was utf_general_ci for this table. I tried different ways and changed this coding to utf_unicode_ci . There is no difference.

**Comment by Jimmi08** [comment]
I had the same problem when I started with version 2 long time ago. There were thread about it in old forum...  very similar to your situation.  
It was related to xampp version, I needed to set something in my.ini... not via phpmyadmin. 
I think it was this:   (the copy from my actual my.ini, I tried your characters, I had no problem)
This should be that:
## UTF 8 Settings
#init-connect=\'SET NAMES utf8\'
collation_server=utf8_unicode_ci
character_set_server=utf8
#skip-character-set-client-handshake

It was so annoying, version 1 never have this problem...  And nobody else didn't see it, because English characters worked fine.  
And if I remember correctly, there were bug in phpmyadmin itself... sorry, I can't help you more.

**Comment by CaMer0n** [comment]
I am unable reproduce this issue in my local test version of v2.x. 
Could you please also check this page to make sure all tables are using the correct encoding:

    e107_admin/db.php?mode=convert_to_utf8

**Comment by Radzi0** [comment]
So i tried to reproduce this issue... and i wasn't able, but... there was something I came across.



Newest e107 github files and fresh instalation. [6 II 2015 - 15:00]
XAMPP Version 5.6.3

Database Utilities Check Charset e107 - utf8_general_ci

I can confirm that after adding `$mySQLcharset = 'utf8';` to e107_config.php, bushes, appeared everywhere. 


I found that there is a problem with displaying polish letters in Menu Name - Admin Area.
'ĄĘĆÓŁŹŻŁ' - set as menu name, and menu title, and i put all the text by Fiedo into the Menu Body, it worked fine in admin area, and page. The only issue i came across was the Menu Name - Admin Area.

![e107](https://cloud.githubusercontent.com/assets/10882348/6081823/c887a96e-ae1c-11e4-8777-ca99a2d3dd61.png)
